README.md

```diff
@@ -1,22 +1,22 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Capstone
 
-> One paragraph statement about the project.
+> In this project i made a capstone wit [this requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)
 
 ![screenshot](./app_screenshot.png)
 
-Additional description about the project and its features.
+This capstone is a presentagion web page for a dog agility contest, with a completely functional
+navigation bar, a mobile menu, and a section created with pure javascript (featured trainers section)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML, CSS & JS
+- Github
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://lisandroseia.github.io/capstone1/)
 
 
 ## Getting Started
```

Comment:
[OPTIONAL]
- The "Getting Started" section is a good place to allow visitors(technical and non-technical) to know how to run your project locally. It would be really nice to have that description set up. 😀 

Here 👇  is an example of a really good "Getting Started" section

![](https://i.imgur.com/C4G2K9e.png)